<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>536</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QGalaxy</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="mnuSettings">
    <property name="title">
     <string>&amp;Настройки</string>
    </property>
    <addaction name="actSettingsOCL"/>
   </widget>
   <widget class="QMenu" name="mnuSim">
    <property name="title">
     <string>&amp;Симуляция</string>
    </property>
    <addaction name="actSimStart"/>
    <addaction name="actSimStop"/>
   </widget>
   <widget class="QMenu" name="mnuGenerate">
    <property name="title">
     <string>&amp;Генерация</string>
    </property>
    <addaction name="actGenSGalaxy"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="mnuSettings"/>
   <addaction name="mnuSim"/>
   <addaction name="mnuGenerate"/>
  </widget>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>Файл</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>120</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Лог</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="vlLog">
     <property name="margin">
      <number>1</number>
     </property>
     <item>
      <widget class="QTextEdit" name="teLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarSettings">
   <property name="windowTitle">
    <string>Настройки</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actSettingsOCL"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Выполнение</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actSimStart"/>
   <addaction name="actSimStop"/>
  </widget>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>В&amp;ыход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actSettingsOCL">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/opencl_icon.png</normaloff>:/images/opencl_icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Настройки OpenCL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actSimStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/run.png</normaloff>:/images/run.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Запустить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actSimStop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Остановить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actGenSGalaxy">
   <property name="text">
    <string>&amp;Спиральная галактика</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
